doc/source/security.md

```diff
@@ -31,25 +31,7 @@ on HTTPS security see the certificates section of [this blog post](https://blog.
 
 ### Set up automatic HTTPS
 
-JupyterHub uses [Let's Encrypt](https://letsencrypt.org/) to automatically create
-HTTPS certificates for your deployment. This will cause your HTTPS certificate
-to automatically renew every few months. To enable this, make the following
-changes to your `config.yaml` file:
-
-1. Specify the two bits of information that we need to automatically provision
-HTTPS certificates - your domain name & a contact email address.
-
-   ```yaml
-   proxy:
-     https:
-       hosts:
-         - <your-domain-name>
-       letsencrypt:
-         contactEmail: <your-email-address>
-   ```
-
-2. Apply the config changes by running `helm upgrade ...`
-3. Wait for about a minute, now your hub should be HTTPS enabled!
+Refer to the Advanced Topics section for Automatic HTTPS with cert-manager and [Let's Encrypt](https://letsencrypt.org/).
```

Comment:
I want to make sure that enabling letsencrypt remains the simplest and first recommended way to enable HTTPS. It shouldn't be moved to advanced topics. This https-enabling config that faces the users should ideally stay the same and the kube-lego->certmanager transition should be hidden from the user as much as possible.

What's the impediment to doing this now?

Reply:
Addressing this is mandatory and in my eyes what needs to be done before we can merge this.

Reply:
Happy to add the simple command back to the security section.